src/resourcepacks/ZippedResourcePack.php

```diff
@@ -105,7 +105,7 @@ public function __construct(string $zipPath){
 		}
 
 		$mapper = new \JsonMapper();
-		$mapper->bExceptionOnUndefinedProperty = true;
+		// $mapper->bExceptionOnUndefinedProperty = true;
```

Comment:
Don't comment code. If you're going to remove it, just remove it.

Reply:
This has been solved in commit [5f495f4](https://github.com/pmmp/PocketMine-MP/pull/4804/commits/5f495f42c99b26a32a743fe3ee503f88e639c3b4).